src/models/modules/bricks/mlp.py

```diff
@@ -0,0 +1,23 @@
+"""TorchOK Multi layer Perceptron module."""
+import torch
+import torch.nn as nn
+
+
+class Mlp(nn.Module):
```

Comment:
Depending on whether DaViT gets merged first, you should take one of the two implementations